db/migrate/20171122121841_change_event_log_user_agent_id_to_integer.rb

```diff
@@ -1,9 +1,15 @@
 class ChangeEventLogUserAgentIdToInteger < ActiveRecord::Migration[5.1]
   def up
+    remove_foreign_key :event_logs, :user_agents
+    change_column :user_agents, :id, "BIGINT"
     change_column :event_logs, :user_agent_id, "BIGINT"
+    add_foreign_key :event_logs, :user_agents
```

Comment:
This table is massive in production (18M rows) and running this migration will lock it for several minutes.  In the past when making changes to it we've used [lhm](https://github.com/soundcloud/lhm/) to avoid locking the tables (see: https://github.com/alphagov/signon/pull/566 and https://github.com/alphagov/signon/pull/567).